src/mscorlib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3119,7 +3232,7 @@ private unsafe bool SendManifest(byte[] rawManifest)
                     dataDescrs[1].Size = (uint)Math.Min(dataLeft, chunkSize);
                     if (m_provider != null)
                     {
-                        if (!m_provider.WriteEvent(ref manifestDescr, null, null, 2, (IntPtr)dataDescrs))
+                        if (!m_provider.WriteEvent(ref manifestDescr, IntPtr.Zero, null, null, 2, (IntPtr)dataDescrs))
```

Comment:
I have confirmed that because we don't set m_rawmanifest we never get to SendManfest and thus passing a null handle is OK here.